src/sections/Timeline.tsx

```diff
@@ -0,0 +1,131 @@
+import type { FC } from 'react'
+
+import { HighlightedTitle } from '../components'
+
+interface InputProps {
+  text: string
+  data: TimelineEntry[]
+}
+
+export interface TimelineEntry {
+  date: string
+  text: string
+}
+
+interface TimlineItemProps {
+  data: TimelineEntry[]
+  isLeft: boolean
+}
+
+interface TimelineTickProps {
+  isLeft: boolean
+  isShown: boolean
+}
+
+const Spacer = () => {
+  return <div className="ml-10 min-h-[50%] min-w-fit" />
+}
+
+const TimelineTick: FC<TimelineTickProps> = ({ isLeft, isShown }) => {
+  return (
+    <>
+      {isShown ? (
+        <>
+          <div className={clsx({ 'mr-10': !isLeft, 'ml-10': isLeft })}>
+            <div
+              className={clsx(
+                'relative top-1/2 w-16 border-t-4 border-clementine',
+                {
+                  'float-right': !isLeft,
+                  'float-left': isLeft,
+                },
+              )}
+            />
+          </div>
+        </>
+      ) : (
+        <></>
+      )}
+    </>
+  )
+}
+
+const TimelineItem: FC<TimlineItemProps> = ({ data, isLeft }) => {
+  return (
+    <>
+      {data.map((item, index) => {
+        return (
+          <div key={index}>
+            {isLeft ? <> </> : <Spacer />}
+            <div className="flex min-h-fit flex-col">
+              <div
+                className={clsx('relative float-right flex w-full flex-row', {
+                  'justify-end': isLeft,
+                  'justify-start': !isLeft,
+                })}
+              >
+                <TimelineTick isLeft={isLeft} isShown={!isLeft} />
+
+                <div className="basis-5/12 rounded-lg bg-darkTeal py-2">
+                  <p className="m-0 text-center text-4xl text-white">
+                    {item.date}
+                  </p>
+                </div>
+                <TimelineTick isLeft={isLeft} isShown={isLeft} />
+              </div>
+              <div className="px-10 py-5">
+                <p
+                  className={clsx('px-10 text-xl text-black', {
+                    'mr-6 text-right': isLeft,
+                    'ml-6 text-left': !isLeft,
+                  })}
+                >
+                  {item.text}
+                </p>
+              </div>
+            </div>
+            {isLeft ? <Spacer /> : <> </>}
+          </div>
+        )
+      })}
+    </>
+  )
+}
+
+const splitTimeline = (
+  data: TimelineEntry[],
+): [TimelineEntry[], TimelineEntry[]] => {
+  const splitData: [TimelineEntry[], TimelineEntry[]] = [[], []]
+  data.forEach((item, index) => {
+    if (index % 2 === 0) {
+      splitData[0].push(item)
+    } else {
+      splitData[1].push(item)
+    }
+  })
+  return splitData
+}
+
+const TimelineSection: FC<InputProps> = ({ text, data }) => {
+  const splitData = splitTimeline(data) ?? [[], []]
+  const splitLeft = splitData[0]
+  const splitRight = splitData[1]
+
+  return (
+    <div className="relative mx-64 min-h-screen border-spacing-1 text-6xl">
```

Comment:
```suggestion
    <div className="relative mx-[10%] min-h-screen border-spacing-1 text-6xl">
```
To make it slightly more responsive :).